openmrs-19/src/test/java/org/motechproject/openmrs19/it/MRSPatientServiceIT.java

```diff
@@ -126,6 +130,45 @@ public void shouldUpdatePatient() throws InterruptedException {
     }
 
     @Test
+    public void shouldUpdatePatientIdentifiers() throws InterruptedException {
+
+        Patient testPatient = patientAdapter.getPatientByMotechId(DEFAULT_CONFIG_NAME, patient.getMotechId());
+
+        String newIdentifiactionNumber = "612";
+        testPatient.getIdentifiers().get(0).setIdentifier(newIdentifiactionNumber);
+
+        synchronized (lock) {
+            patientAdapter.updatePatientIdentifiers(DEFAULT_CONFIG_NAME, testPatient);
+
+            lock.wait(60000);
+        }
+        Patient updated = patientAdapter.getPatientByUuid(DEFAULT_CONFIG_NAME, testPatient.getUuid());
+
+        assertEquals("Old Identification Number", updated.getIdentifiers().get(0).getIdentifierType().getName());
+        assertEquals(newIdentifiactionNumber, updated.getIdentifiers().get(0).getIdentifier());
+    }
+
+    @Test
+    public void shouldAddPatientIdentifiers() throws InterruptedException {
+
+        Patient testPatient = new Patient();
```

Comment:
As far as I understand to test this, you will have to add new identifier type to the OpenMRS docker. Don't do this in this PR, just omit this test. We will add an appropriate identifier type to OpenMRS docker while creating IT test for task integration